src/Lean/Elab/Import.lean

```diff
@@ -42,4 +42,12 @@ def printImports (input : String) (fileName : Option String) : IO Unit := do
     let fname ← findOLean dep.module
     IO.println fname
 
+@[export lean_print_import_srcs]
+def printImportSrcs (input : String) (fileName : Option String) : IO Unit := do
+  let sp ← initSrcSearchPath
+  let (deps, _, _) ← parseImports input fileName
```

Comment:
Should this be `parseImports'` for speed?

Reply:
@eric-wieser Probably, but as `--deps` does not use it either, that refactor seems best left for a separate PR.